bin/ds9_itgcoords

```diff
@@ -25,11 +25,25 @@ from coords.gratings import TGPixlib
 from dax.utils import xpaget, xpaset_p
 
 def init_pixlib(xpa):
-    'Setup pixlib and various xforms from header info'
-    from pycrates import read_file
-    infile = xpaget(xpa, "file").strip()
-    tab = read_file(infile)
-    keywords = {x: tab.get_key_value(x) for x in tab.get_keynames()}
+    '''Setup pixlib and various xforms from header info
+    
+    Turns out this is faster than sending file name to pycrates
+    '''
+    _keys = ["instrume", "detnam", "grating", "sim_x", "sim_y", "sim_z",
+             "dy_avg", "dz_avg", "dth_avg", "ra_pnt", "dec_pnt", "roll_pnt",
+             "ra_nom", "dec_nom"]
+    
+    keywords = {}
+    for kk in _keys:
+        kk_upper = kk.upper()
+        keyval = xpaget(xpa, f"fits header keyword {kk_upper}").strip()
+        if keyval == "":
+            raise ValueError("Unable to find the value for the '{kk_upper}' keyword")
```

Comment:
Should this be (not particularly important)

```suggestion
            raise ValueError(f"Unable to find the value for the '{{{kk_upper}}}' keyword")
```

As a check I have enough brackets

```
>>> kk_upper = "BOB"
>>> f"blah '{{{kk_upper}}}' blah"
"blah '{BOB}' blah"
```

Reply:
sorry i missed that i missed the 'f' which is what I wanted